### PR TITLE
Disable build-where-somebody-stands serverside

### PR DESCRIPTION
**Pull request by appgurueu**
Counters pillaring.

### CODE REVIEW

File: mods/pvp/antipillar/init.lua [hunk 1]
```diff
@@ -0,0 +1,180 @@
+local visualize_box
```
Comment (Panquesito7) [inline mods/pvp/antipillar/init.lua:1]:
```suggestion
-- Capture The Flag mod: antipillar

local visualize_box
```

File: mods/pvp/antipillar/test.lua [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+local function visualize_box(pos, box)
+    local obj = minetest.add_entity(vector.add(pos, {x = (box[1] + box[4]) / 2, y = (box[2] + box[5]) / 2, z = (box[3] + box[6]) / 2}), test_box)
+    obj:set_properties{
+        infotext = "Collisionbox: " .. minetest.write_json(box),
+        visual_size = {x = box[4] - box[1], y = box[5] - box[2], z = box[6] - box[3]}
+    }
+end
+
+return visualize_box
```
Comment (Panquesito7) [inline mods/pvp/antipillar/test.lua:32]:
```suggestion
return visualize_box

```

File: mods/pvp/antipillar/mod.conf [hunk 3]
```diff
@@ -0,0 +1,2 @@
+name = antipillar
+description = Prevents pillaring by not allowing players to build where they stand
```
Comment (Panquesito7) [inline mods/pvp/antipillar/mod.conf:2]:
```suggestion
description = Prevents pillaring by not allowing players to build where they stand

```

File: mods/pvp/antipillar/init.lua [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+            for _, collisionbox in pairs(get_node_collisionboxes(newnode, pos)) do
+                if visualize_box then visualize_box(pos, collisionbox) end
+                if aabb_collision(collisionbox_player, collisionbox, diff) then
+                    minetest.set_node(pos, oldnode)
+                    return true
+                end
+            end
+        end
+    end
+end)
```
Comment (Panquesito7) [inline mods/pvp/antipillar/init.lua:180]:
```suggestion
end)

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by appgurueu** [comment]
@Panquesito7 I don't like that code style. The "Ctf mod" comment adds almost nothing, the additional line breaks are unneeded on modern systems. Is this the accepted code style?

File: mods/pvp/antipillar/init.lua [hunk 5]
```diff
@@ -0,0 +1,180 @@
+local visualize_box
+-- uncomment for testing:
+visualize_box = loadfile(minetest.get_modpath(minetest.get_current_modname()) .. "/test.lua")()
```
Comment (appgurueu) [inline mods/pvp/antipillar/init.lua:3]: Currently uncommented, I will comment this again on merge

**Comment by appgurueu** [comment]
Also see https://content.minetest.net/packages/LMD/disable_build_where_they_stand/

**Pull request closed without merging**